pagopa/pn-frontend
# PN-10951-fe-api-exchange-qr-code-pf-pg

## Short Description
Migrate the exchangeQrCode API to the BFF's API.

## List of Changes Proposed in This Pull Request
- Generated models from the BFF OpenAPI.
- Updated `exchangeNotificationQrCode()` with the BFF's API.
- Updated related tests.

## How to Test
Start PF/PG and verify that everything works as intended by checking if the QR Code of the AAR displays the notification details correctly.
